app/components/IndicatorDevelopment/IndicatorDevelopmentContent.js

```diff
@@ -6,12 +6,27 @@ import IndicatorDevelopmentContentHeader from './IndicatorDevelopmentContentHead
 import IndicatorDevelopmentList from './IndicatorDevelopmentList';
 import NoDataMessage from '../NoDataMessage';
 import { containerPadding } from '../../utils/responsive_util';
+import Rating from '../../models/Rating';
 
 class IndicatorDevelopmentContent extends Component {
   static contextType = LocalizationContext;
+  constructor(props) {
+    super(props);
 
-  state = {
-    headerHeight: 0,
+    this.state = {
+      headerHeight: 0,
+      hasRating: Rating.getAll(props.scorecardUuid).length > 0 ? true : false
+    }
+  }
+
+  componentDidMount() {
+    this.focusListener = this.props.navigation.addListener("focus", async () => {
+      this.setState({ hasRating: Rating.getAll(this.props.scorecardUuid).length > 0 ? true : false });
```

Comment:
@limkimsan can we move Rating.getAll(this.props.scorecardUuid).length > 0 into Rating.has(this.props.scorecardUuid) so it will make code here even readable

Reply:
Noted bong, I will update it to Rating.has(this.props.scorecardUuid).